corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -371,9 +371,10 @@ public void onError(Throwable throwable) {
         }
 
         StreamDumper streamDumper = new StreamDumper();
-        List<TableSchema<Message, Message, Message>> tablesOfInterest = new ArrayList<>();
-        tablesOfInterest.add(streamDumper.getTableSchema());
-        store.subscribe(streamDumper, namespace, tablesOfInterest, null);
+        List<String> tablesOfInterest = Collections.singletonList(tableName);
+        String streamTag = ExampleTableName.getDescriptor().getOptions()
+                .getExtension(CorfuOptions.tableSchema).getStreamTag(0);
+        store.subscribeListener(streamDumper, namespace, streamTag, tablesOfInterest, null);
```

Comment:
where are we writing to the table?  is it not required?

Reply:
The writing is in loadTable()